experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts

```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
 	}
 	private maybeTree?: ISharedTree = undefined;
 	private maybeAppState?: AppState = undefined;
```

Comment:
My personal preference is to type these as non-optional but unioned with undefined as I think it communicates the two-state nature better?  But might just be an opinion.

```suggestion
	private _tree: ISharedTree | undefined;
	private _appState: AppState | undefined;
```